tests/wp-unit/tests/include/posts/Bulk_Delete_Posts_By_Post_TagTest.php

```diff
@@ -1,256 +0,0 @@
-<?php
```

Comment:
@anandB000 

Any specific reason for removing all these tests?

You could have simply refactored them for your next task.

Reply:
I will include it when start delete by tag task

Reply:
Okay. But in future don't remove files like this since this will mess up git history.

Reply:
Okay